.github/workflows/build-packages.yml

```diff
@@ -0,0 +1,74 @@
+name: "Debian Package Builder"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  build-pkg:
+    name: Build debian packages
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      # # We need the whole history so we can make a pretty version
+      # with:
+      #   fetch-depth: 0
+    - name: Install required tools
+      run: |
+        sudo apt update
+        sudo apt install -y acl zip unzip apache2 php php-fpm php-gd \
+                            php-cli php-intl php-mbstring php-mysql php-curl php-json \
+                            php-xml php-zip ntp make sudo debootstrap \
+                            libcgroup-dev lsof php-cli php-curl php-json php-xml \
+                            php-zip procps gcc g++ default-jre-headless \
+                            default-jdk-headless ghc fp-compiler autoconf automake bats \
+                            python3-sphinx python3-sphinx-rtd-theme rst2pdf fontconfig \
+                            python3-yaml latexmk
```

Comment:
Probably we should install the Build-Deps that the packaging specifies instead of duplicating it here.